lib/sycamore/sycamore/llms/prompts/default_prompts.py

```diff
@@ -48,6 +50,10 @@ def _execute_instruct(self, model, **kwargs) -> str:
         lm += gen(name=self.var_name)
         return lm[self.var_name]
 
+    def _execute_completion(self, model, **kwargs) -> str:
```

Comment:
we should probably add tests at some point but this is fine for now since it'll break integrations and be detected